docs/reference.rst

```diff
@@ -670,6 +670,7 @@ Command Line Tool
     extract               Extract the bundle content
     install               Install a bundle
     info                  Show file information
+    mount                 Mount RAUC bundle
```

Comment:
```suggestion
    mount                 Mount a bundle
```
This would be consistent with the other command's descriptions.

Reply:
Changed.